app/assets/src/components/visualizations/heatmap/Heatmap.js

```diff
@@ -809,8 +862,8 @@ export default class Heatmap {
       .enter()
       .append("g")
       .attr("class", cs.rowLabel)
-      .on("mousein", this.options.onRowLabelMouseIn)
-      .on("mouseout", this.options.onRowLabelMouseOut);
+      .on("mouseover", this.handleRowLabelMouseOver)
```

Comment:
You want `mousein` here instead of `mouseover - or in the next pr - so it does not continuously trigger.

Reply:
Good point. I switched to mouseover from mousein because it did not bubble up. When I hovered over the text inside the box the tooltip disappeared. Do you have any solutions for that?

Reply:
Was it triggering a mouse out as you were moving? 

Typically, on `mousein` you should create the tooltip and save the position and clear it on `mouseout`. Mouse over should change anything...

Reply:
The problem is that the mouseout is firing when hovering over the text label inside the g element.